client/src/store/modules/auth/actions.js

```diff
@@ -0,0 +1,41 @@
+import * as actions from './types/actions';
+import * as mutations from './types/mutations';
+import authService from '@/services/auth/authService';
+
+export default {
+    [actions.SIGN_IN]: async ({ commit }, loginData) => {
+        try {
+            const loginResponse = await authService.signIn(loginData);
+            commit(mutations.USER_LOGIN, {
+                accessToken: loginResponse.access_token,
+                tokenType: loginResponse.token_type,
+                expiresIn: loginResponse.expires_in
+            });
+        } catch (error) {
```

Comment:
if you don't prepare the error, you don't have to wrap logic with try..catch, it is redundant

Reply:
it relates for all actions